math/lcm.go

```diff
@@ -0,0 +1,49 @@
+// Formula from https://en.wikipedia.org/wiki/Least_common_multiple
```

Comment:
Start this comment with the package name that you have chosen.

```suggestion
// lcm Formula from https://en.wikipedia.org/wiki/Least_common_multiple
```